tests/robot-tests/tests/general_public/prod_only/redirects.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Resource            ../libs/public-common.robot
+
+Suite Setup         user opens the browser
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          GeneralPublic    Local    Dev    Test    Preprod
```

Comment:
Hey Sam, this combo of tags will unfortunately not run against Prod here.  For this test in this folder, I'd just include the "GeneralPublic" and "Prod" tags.